hedera-node/hedera-smart-contract-service-impl/src/main/java/com/hedera/node/app/service/contract/impl/infra/HevmTransactionFactory.java

```diff
@@ -295,6 +296,7 @@ private void assertValidCreation(@NonNull final ContractCreateTransactionBody bo
 
     private Bytes initcodeFor(@NonNull final ContractCreateTransactionBody body) {
         if (body.hasInitcode()) {
+            validateTrue(body.initcode().length() > 0, CONTRACT_BYTECODE_EMPTY);
```

Comment:
I actually like this `validateTrue` better than the previously seen `ContractCreateTransitionLogic` L353 `validateFalse`.  `validateFalse` just doesn't work, generally, with my brain: It is easier (for me) to immediately grok validate TRUE **not** empty than validate FALSE **is** empty.

I wasn't going to mention it up there, but since you haven't been _consistent_ I consider it fair game now.

Reply:
This bothered me too but if you noticed, I was being consistent with the check that returns `CONTRACT_FILE_EMPTY` within in the same file while not being consistent across mono vs modular code base.  I think this is the less of two evils as the mono code will eventually be deprecated?

Reply:
May post modularization we make them all validateTrue 🤔